invenio_github/api.py

```diff
@@ -245,14 +246,31 @@ def _sync_hooks(self, repos, asynchronous=True):
             db.session.commit()
             sync_hooks_task.delay(self.user_id, repos)
 
+    def _valid_webhook(self, url):
+        """Check if webhook url is valid.
+
+        The webhook url is valid if it has the same host as the configured webhook url.
+
+        :param str url: The webhook url to be checked.
+        :returns: True if the webhook url is valid, False otherwise.
+        """
+        # Check if webhook is valid
+        if not url or not self.webhook_url:
```

Comment:
```suggestion
        if not url:
```

- nit: the comment is a bit redundant
- nit: I think there's no chance that we don't have a `self.webhook_url`, so no point to check again